package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "foreman_leapp",
+  "version": "0.0.1",
+  "description": "Plugin for RHEL upgrades using Foreman.",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "node node_modules/.bin/jest webpack",
+    "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/oamg/foreman_leapp.git"
+  },
+  "author": "LEAPP team + The Foreman team",
+  "license": "GPL-3.0",
+  "bugs": {
+    "url": "https://projects.theforeman.org"
```

Comment:
can you add the project name, e.g:
"url": "http://projects.theforeman.org/projects/foreman_leapp/issues"